web3swift/src/Client/Models/EthereumTransaction.swift

```diff
@@ -132,7 +132,7 @@ public struct EthereumTransaction: EthereumTransactionProtocol, Equatable, Codab
     }
 }
 
-struct SignedTransaction {
+public struct SignedTransaction {
```

Comment:
Please make `transaction` and `hash` public too

Reply:
Fixed in 0aa64a5d0262ddf15d8c1ab3c6cc85d88fc360de